man/systemd.special.xml

```diff
@@ -285,15 +286,17 @@
         <term><filename>hibernate.target</filename></term>
         <listitem>
           <para>A special target unit for hibernating the system. This
-          pulls in <filename>sleep.target</filename>.</para>
+          pulls in <filename>sleep.target</filename> and
+          <filename>wakeup.target</filename>.</para>
```

Comment:
I have indeed seen someone replace systemd-hibernate.service.  I'm not sure about that argument.

Hmm, suspend.target is ordered After=systemd-suspend.service.  So I think this approach would break down for sleep.target.  Does anyone know why that is?